doc/changelog/04-tactics/13381-bfs_eauto.rst

```diff
@@ -1,5 +1,5 @@
 - **Deprecated:**
-  Undocumented :n:`eauto @int_or_var @int_or_var` syntax in favor of new ``bfs eauto``.
+  Undocumented :n:`eauto @nat_or_var @nat_or_var` syntax in favor of new ``bfs eauto``.
   Also deprecated 2-integer syntax for ``debug eauto`` and ``info_eauto``.
   (Use ``bfs eauto`` with the :flag:`Info Eauto` or :flag:`Debug Eauto` flags instead.)
```

Comment:
You could put a few `:tacn:` in this changelog entry (not an important suggestion).